core/src/main/java/org/infinispan/util/logging/events/EventLogger.java

```diff
@@ -19,7 +20,7 @@
  * @author Tristan Tarrant
  * @since 8.2
  */
-public interface EventLogger {
+public interface EventLogger extends Listenable {
```

Comment:
I don't think we need to extend the `EventLogger` to be `Listenable`, instead we can just register the `Loggable` listener to the `EmbeddedCacheManager` instance as event logs are already globally scoped.

Reply:
Do you mean we register directly into the `EmbeddedCacheManager` instead of the `EventLogger`?

Reply:
Exactly

Reply:
Moved some stuff to the `CacheManagerNotifier`, so was able to delete that notifier I've created.